### PR TITLE
Cpr 395 consume fifo

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -19,7 +19,7 @@ configurations {
 
 dependencies {
   implementation("uk.gov.justice.service.hmpps:hmpps-kotlin-spring-boot-starter:1.0.2")
-  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:4.0.1")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:4.2.0-beta-4")
```
Comment (markreesmoj) [inline build.gradle.kts:22]: Not yet published so this will fail to build

File: build.gradle.kts [hunk 2]
```diff
@@ -57,6 +57,7 @@ java {
 }
 
 repositories {
+  mavenLocal()
```
Comment (markreesmoj) [inline build.gradle.kts:60]: oops

File: build.gradle.kts (hunk 1 above)
Comment (markreesmoj) [reply build.gradle.kts]: published now

File: build.gradle.kts (hunk 2 above)
Comment (markreesmoj) [reply build.gradle.kts]: now published

**Review by H-Iwanejko — DISMISSED** [review]

File: src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtEventFIFOListener.kt [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+const val CPR_COURT_EVENTS_QUEUE_CONFIG_KEY = "cprcourteventsqueue"
+
+@Component
+@Profile(value = ["preprod", "test"])
+class CourtEventFIFOListener(
+  val objectMapper: ObjectMapper,
+  val telemetryService: TelemetryService,
+) {
+  @SqsListener(CPR_COURT_EVENTS_QUEUE_CONFIG_KEY, factory = "hmppsQueueContainerFactoryProxy")
```
Comment (markreesmoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtEventFIFOListener.kt:29]: Need to add the config for the queue here

**Review by sameermoj — APPROVED** [review]
LGTM

**Pull request merged**